int32 posA
int32 posB
float64 velAY
float64 velBX
float64 trqA
float64 trqB
float64 loadA
float64 loadB
float64 voltM
float64 voltC
int32 inp
int32 outp
int32 incontrol
int32 outcontrol
int32 warning
int32 alarm

